BR FAQ: add a warning about multi br importing (#6263)
pingcap/docs: br/use-br-command-line-tool.md

> - Where each peer is scattered to during restore is random. We don't know in advance which node will read which file.
>
> These can be avoided using shared storage, for example mounting an NFS on the local path, or using S3. With network storage, every node can automatically read every SST file, so these caveats no longer apply.
> 

> > Also, note that you can only run one restore operation for a single cluster at the same time. Otherwise, unexpected behaviors might occur. For details, see [FAQ](/br/backup-and-restore-faq.md#can-i-use-multiple-br-command-lines-at-the-same-time-to-restore-the-data-of-a-single-cluster).

Author: ```suggestion
> Also, note that you can only run one restore operation for a single cluster at the same time. Otherwise, unexpected behaviors might occur. For details, see [FAQ](/br/backup-and-restore-faq.md#can-i-use-multiple-br-processes-at-the-same-time-to-restore-the-data-of-a-single-cluster).
```

> > Also, note that you can only run one restore operation for a single cluster at the same time. Otherwise, unexpected circumstances might occur. For details, see [FAQ](/br/backup-and-restore-faq.md#can-i-use-multiple-br-command-lines-at-a-single-time-to-restore-the-data-of-a-single-cluster).

Author: ```suggestion
> Also, note that you can only run one restore operation for a single cluster at the same time. Otherwise, unexpected circumstances might occur. For details, see [FAQ](/br/backup-and-restore-faq.md#can-i-use-multiple-br-command-lines-at-the-same-time-to-restore-the-data-of-a-single-cluster).
```

> > Also, note that you can only run one restore operation for a single cluster at the same time. Otherwise, unexpected circumstances might occur. For details, see [FAQ](/br/backup-and-restore-faq.md#can-i-use-multiple-br-command-lines-at-the-same-time-to-restore-the-data-of-a-single-cluster).

Reviewer: ```suggestion
> Also, note that you can only run one restore operation for a single cluster at the same time. Otherwise, unexpected behaviors might occur. For details, see [FAQ](/br/backup-and-restore-faq.md#can-i-use-multiple-br-command-lines-at-the-same-time-to-restore-the-data-of-a-single-cluster).
```
